test/CMakeLists.txt

```diff
@@ -398,7 +398,7 @@ celeritas_add_device_test(celeritas/mat/Material)
 #-------------------------------------#
 # Phys
 set(CELERITASTEST_PREFIX celeritas/phys)
-celeritas_add_test(celeritas/phys/CutoffParams.test.cc)
+celeritas_add_test(celeritas/phys/CutoffParams.test.cc ${_needs_root})
```

Comment:
Since some of the cutoff tests doesn't require ROOT, revert this and wrap the root test name with the `TEST_IF_CELERITAS_USE_ROOT` macro.

Reply:
```c++
#define CutoffParamsImportTest TEST_IF_CELERITAS_USE_ROOT(CutoffParamsImportTest)
```